### ISSUE TITLE
Security Hotspots not visible in Development Studio

### ISSUE THREAD

**Issue by lievencardoen** [Riverside-Software/sonar-openedge#1147]
Hi Gilles,

Is there a way to see Security Hotspots in the OpenEdge IDE? Or am I doing something wrong.

Security Hotspots reported by Sonar and visible in the Sonar Web UI are not reported in my Development Studio...

Kind regards, Lieven

**Comment by gquerret** [comment]
I don't use that often, but you can use the "Open in IDE" button from SonarQube:
![image](https://github.com/user-attachments/assets/3583c621-3ca0-4b6e-8653-3422cd34a50b)

Then select the right IDE:
![image](https://github.com/user-attachments/assets/454850c5-ae88-4779-ab40-3e132fc50728)

This opens the file in PDSOE:
![image](https://github.com/user-attachments/assets/cf8087da-cc07-4aa4-8d88-7fc190786881)

This is the only way to do that in Eclipse: https://docs.sonarsource.com/sonarlint/eclipse/using-sonarlint/security-hotspots/#hotspot-analysis


Execution of security hotspots is currently not available in SonarLint for VS Code (for ABL code), it will be added soon.

**Comment by lievencardoen** [comment]
Hi Gilles, so you can see them in Development Studio, but only if you do it from Sonar and Open in IDE?

There's no way to do that scan in Development Studio? Am I correct?

Thx, Lieven

**Comment by lievencardoen** [comment]
Do you need to configure Sonar somehow to let it open in an IDE? 

![{323F7219-DC52-4D7B-84B1-1034377B667C}](https://github.com/user-attachments/assets/5d24569b-6c5e-4774-b49b-0b16f63b0967)

**Comment by gquerret** [comment]
> There's no way to do that scan in Development Studio? Am I correct?

Yes, you can't do the scan from PDSOE, and I don't think SLint Eclipse will ever support that. It will be possible in VS Code.

> Do you need to configure Sonar somehow to let it open in an IDE?

Which version of the SonarLint plugin do you use ? My test was done with SonarLint 10.4.3202.

**Comment by lievencardoen** [comment]
SonarLint for Eclipse 9.3.3002.

Outdated?

**Comment by gquerret** [comment]
Depending on your version of PDSOE, you can update to 10.3 or 10.4: [https://github.com/Riverside-Software/sonar-openedge/wiki/SonarLint-for-Eclipse#download-and-install](https://github.com/Riverside-Software/sonar-openedge/wiki/SonarLint-for-Eclipse#download-and-install)
